test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -207,6 +207,63 @@ def forward(self, input):
         input = torch.randn(3, 4, requires_grad=True)
         self.run_model_test(model, train=False, batch_size=0, input=input)
 
+    def test_onnx_export_with_parameter_renaming(self):
+        class SimpleFcNet(nn.Module):
+            def __init__(self):
+                super(SimpleFcNet, self).__init__()
+                self.fc1 = nn.Linear(5, 10)
+
+            def forward(self, input):
+                return self.fc1(input)
+
+        model = SimpleFcNet()
+        input = torch.randn(7, 5)
+        output = model(input)
+
+        import io
+        f = io.BytesIO()
+        from torch.onnx import ExportTypes
```

Comment:
Nit: lift this import to the beginning of the file.

Reply:
Done.